src/main/java/DateAndTime.java

```diff
@@ -0,0 +1,51 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Customised Data and Time class to store and print out appropriate
+ * dates and times for tasks.
+ */
+public class DateAndTime {
+
+    private final LocalDate date;
+    private final LocalTime time;
+
+    /**
+     * Construct an date and time object for tasks without specified time.
```

Comment:
Perhaps the first word should be "Constructs", as in a verb ending with an "s"? I noticed this similar issue for a few other Javadoc comments but most of them do follow this coding standard rule, which is great!